### DISCUSSION TITLE
How to use vars prop

### DISCUSSION THREAD

**Discussion by PPHelios** [mantinedev/mantine#6194] [Q&A] [answered]
In production "--tabs-color" in light theme is always transparent and not affected with background-color, I'm trying to follow the documentation and change "--tabs-color" directly from the component using vars prop but I don't know how to use it.

**Comment by rtivital** [comment]
https://mantine.dev/styles/styles-api/#components-css-variables

**Reply by PPHelios** [reply]
thanks but for some reason  in production background-color in light mode only is always overridden by
 .m_87cf2631 {background-color: transparent;}
but it works fine in dev and dark mode.

**Reply by rtivital** [reply, marked as answer]
https://mantine.dev/styles/mantine-styles/#css-layers
